Example/Example/AppFlowController.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+import FlowControllerKit
+
+class AppFlowController {
+
+    let storyboard = UIStoryboard(name: "Main", bundle: nil)
+
+    var valueVC: ValueViewController!
+    var navigationController: UINavigationController
+
+    init(window: UIWindow) {
+        valueVC = storyboard.instantiate(ValueViewController.self)
+        navigationController = UINavigationController(rootViewController: valueVC)
+
+        window.rootViewController = navigationController
+        window.makeKeyAndVisible()
+    }
+
+    func start() {
+        let changeValueVC = storyboard.instantiate(ChangeValueViewController.self)
+        let modalVC = storyboard.instantiate(ModalViewController.self)
+
+        valueVC.changingValue = valueVC.showing(changeValueVC, preparation: changeValueVC.setup(value:))
+        valueVC.presentModal = valueVC.presenting(modalVC)
+
+        changeValueVC.changedValue = valueVC.setup(value:)
```

Comment:
I think you've missed to type a value here? Or is it some syntactic sugar I don't know about? Either way it would be clear if something were typed there